### ISSUE TITLE
An error occurred while installing.

### ISSUE THREAD

**Issue by taji-taji** [jakeheis/Flock#2]
Thank you for nice project.

I attempted to install Flock by brew, but I couldn’t.

```
$ brew install jakeheis/repo/flock
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/core).
No changes to formulae.

==> Installing flock from jakeheis/repo
==> Downloading https://github.com/jakeheis/FlockCLI/archive/0.0.1.tar.gz
Already downloaded: /Users/tajika/Library/Caches/Homebrew/flock-0.0.1.tar.gz
==> swift build -c release
Last 15 lines from /Users/tajika/Library/Logs/Homebrew/flock/01.swift:
2016-11-12 18:35:39 +0900

swift build -c release

swift-build: error: invalid inferred toolchain: could not find `clang` at expected path /private/tmp/flock-20161112-90931-igoj2t/FlockCLI-0.0.1/clang

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
https://github.com/jakeheis/homebrew-repo/issues
```

Next, I tried to install Flock manually, but an error occurred at `swift build -c release`.

```
$ git clone https://github.com/jakeheis/FlockCLI
Cloning into 'FlockCLI'...
remote: Counting objects: 305, done.
remote: Compressing objects: 100% (8/8), done.
remote: Total 305 (delta 3), reused 0 (delta 0), pack-reused 297
Receiving objects: 100% (305/305), 45.89 KiB | 0 bytes/s, done.
Resolving deltas: 100% (197/197), done.
Checking connectivity... done.

$ cd FlockCLI

$ swift build -c release
Cloning https://github.com/jakeheis/SwiftCLI
HEAD is now at f5ffa65 Remove alias function
Resolved version: 2.0.2
Cloning https://github.com/onevcat/Rainbow
HEAD is now at 4feb43c Bump version to 2.0.1
Resolved version: 2.0.1
Cloning https://github.com/nvzqz/FileKit
HEAD is now at 28767c8 Prepare 4.0.0 release
Resolved version: 4.0.0
Cloning https://github.com/jakeheis/Spawn
HEAD is now at d2a245f Wait
Resolved version: 0.0.4
error: the package has an unsupported layout, unexpected source file(s) found: /Users/tajika/FlockCLI/Packages/FileKit-4.0.0/Tests/FileKitTests.swift
fix: move the file(s) inside a module
```

Finally, I moved `Tests/FileKitTests.swift` temporally, `swift build` is success.

```
$ mv /Users/tajika/FlockCLI/Packages/FileKit-4.0.0/Tests/FileKitTests.swift /tmp/

$ swift build -c release
Compile Swift Module 'Spawn' (1 sources)
Compile Swift Module 'Rainbow' (11 sources)
Compile Swift Module 'FileKit' (33 sources)
Compile Swift Module 'SwiftCLI' (18 sources)
/Users/tajika/FlockCLI/Packages/FileKit-4.0.0/Sources/Path.swift:1074:17: warning: 'ExpressibleByStringInterpolation' is deprecated: it will be replaced or redesigned in Swift 4.0.  Instead of conforming to 'ExpressibleByStringInterpolation', consider adding an 'init(_:String)'
extension Path: ExpressibleByStringInterpolation {
                ^
Compile Swift Module 'FlockCLI' (13 sources)
Linking ./.build/release/FlockCLI
```

I do not understand this error properly, but what does this mean?

---

[My Environment]

Apple Swift version 3.0.1 (swiftlang-800.0.58.6 clang-800.0.42.1)
macOS Sierra 10.12.1

**Comment by jakeheis** [comment]
The first error appears to be an issue with your Swift installation, not with Flock, as the compiler is having trouble finding `clang` which is a problem independent of Flock. With regards to the second error, that may have something to do with Swift 3.0.1 vs 3.0 -- I'll update my Swift version and investigate further. Thanks for reporting.

**Comment by taji-taji** [comment]
OK, I see. Thanks!

**Comment by krystofbe** [comment]
I'm getting the exact same error @jakeheis 

```
==> Installing flock from jakeheis/repo
==> Downloading https://github.com/jakeheis/FlockCLI/archive/0.0.1.tar.gz
Already downloaded: /Users/krystof/Library/Caches/Homebrew/flock-0.0.1.tar.gz
==> swift build -c release
Last 15 lines from /Users/krystof/Library/Logs/Homebrew/flock/01.swift:
2016-11-23 18:36:53 +0100

swift build -c release

swift-build: error: invalid inferred toolchain: could not find `clang` at expected path /private/tmp/flock-20161123-63174-4hqvdl/FlockCLI-0.0.1/clang
```

swift -version outputs
```
Apple Swift version 3.0.1 (swiftlang-800.0.58.6 clang-800.0.42.1)
Target: x86_64-apple-macosx10.9
```

macOS Sierra 10.12.1

Any Ideas?

**Comment by jakeheis** [comment]
Both the clang error and FileKit error should now be fixed. @taji-taji and @krystofbe when you get a chance, please confirm that it now works for you!

**Comment by krystofbe** [comment]
@jakeheis Thanks for fixing the error.
Unfortunately I'm getting a new error now:

`==> Installing flock from jakeheis/repo
==> Downloading https://github.com/jakeheis/FlockCLI/archive/0.0.3.tar.gz
Already downloaded: /Users/krystof/Library/Caches/Homebrew/flock-0.0.3.tar.gz
==> unset CC; swift build -c release
Last 15 lines from /Users/krystof/Library/Logs/Homebrew/flock/01.unset:
Resolved version: 0.0.4
Compile Swift Module 'Spawn' (1 sources)
Compile Swift Module 'Spectre' (8 sources)
Compile Swift Module 'Rainbow' (11 sources)
Compile Swift Module 'SwiftCLI' (18 sources)
Compile Swift Module 'PathKit' (1 sources)
Compile Swift Module 'FlockCLI' (13 sources)
/private/tmp/flock-20161202-10297-1wkq62l/FlockCLI-0.0.3/Sources/HelpCommand.swift:13:7: error: type 'HelpCommand' does not conform to protocol 'HelpCommand'
class HelpCommand: SwiftCLI.HelpCommand, FlockCommand {
      ^
SwiftCLI.HelpCommand:3:16: note: protocol requires property 'availableCommands' with type '[Command]'; do you want to add a stub?
    public var availableCommands: [Command] { get set }
               ^
<unknown>:0: error: build had 1 command failures
swift-build: error: exit(1): /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swift-build-tool -f /private/tmp/flock-20161202-10297-1wkq62l/FlockCLI-0.0.3/.build/release.yaml`

Am i doing something wrong? I'll try to install manually

**Comment by krystofbe** [comment]
Manual building works btw

**Comment by jakeheis** [comment]
Thanks for reporting this, fixed now in new version `0.1.0`
